das-dashboard/backend/services/config_services.py

```
@@ -98,14 +101,18 @@ def _scp_config_sync(self, ip: str, nested: dict) -> str:
 
     async def load_config(self, nested_config: dict) -> dict:
         if not isinstance(nested_config, dict):
-            raise ValueError("Configuration must be a JSON object.")
+            raise ConfigurationFileLoadError("Configuration must be a JSON object.")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)